web/src/message_edit_history.ts

```diff
@@ -2,37 +2,41 @@ import $ from "jquery";
 import assert from "minimalistic-assert";
 import {z} from "zod";
 
-import render_message_edit_history from "../templates/message_edit_history.hbs";
-import render_message_history_modal from "../templates/message_history_modal.hbs";
+import render_message_history from "../templates/message_edit_history.hbs";
```

Comment:
Since `message_edit_history` is the name of the partial, why has it been changed to `render_message_history` here? We tend to keep those things consistent.

Reply:
Yeah I noted this as well; fixing while merging but please be more careful to follow naming conventions.

Reply:
Thank you for taking care of it,